worker/src/main/java/org/apache/celeborn/service/deploy/worker/storage/CreditStreamManager.java

```diff
@@ -253,7 +253,7 @@ public int getActiveMapPartitionCount() {
     return activeMapPartitions.size();
   }
 
-  protected class StreamState {
+  protected static class StreamState {
```

Comment:
Should this be `protected static` or just `static`?

Reply:
@venkata91, this just follow `ClassCanBeStatic` rule. No need to remove protected.